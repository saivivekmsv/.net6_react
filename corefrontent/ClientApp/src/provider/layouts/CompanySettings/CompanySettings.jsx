import React from "react";
import { CompanySettingsContainer } from "../../containers";

const CompanySettings = (props) => {
  return <CompanySettingsContainer {...props} />;
};

export default CompanySettings;
