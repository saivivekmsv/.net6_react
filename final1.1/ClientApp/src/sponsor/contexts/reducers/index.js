export * from "./app-layout";
export * from "./manage-company";
export * from "./create-plan";
export * from "./manage-eligibility";
export * from "./manage-census";
export * from "./manage-payroll";
export * from "./manage-maintenance";
export * from "./manage-report";
export * from "./manage-mapper";
