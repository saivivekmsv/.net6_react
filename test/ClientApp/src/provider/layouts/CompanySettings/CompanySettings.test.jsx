import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import CompanySettings from "./CompanySettings";
import { ManageCompanyProvider } from "../../contexts";

test("renders learn react link", () => {
  const wrapper = render(
    <ManageCompanyProvider>
      <Router>
        <CompanySettings />
      </Router>
    </ManageCompanyProvider>
  );
  expect(wrapper.baseElement).toMatchSnapshot();
});
