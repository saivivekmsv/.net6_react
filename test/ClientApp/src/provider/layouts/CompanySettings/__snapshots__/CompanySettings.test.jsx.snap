// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<body>
  <div>
    <form
      class="h-100"
    >
      <div
        class="d-flex flex-column manage-company h-100"
      >
        <div
          class="d-flex layout-header align-items-center"
        >
          <div>
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="breadcrumb"
              >
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/home"
                    >
                      Home
                    </a>
                  </a>
                </li>
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/companys"
                    >
                      Manage Company
                    </a>
                  </a>
                </li>
              </ol>
            </nav>
            <h5 />
          </div>
          <div
            class="d-flex align-items-center justify-content-center tab-wrapper"
          />
          <div
            class="ml-auto"
          >
            <div>
              <a
                href="/companys"
              >
                <button
                  class="btn btn-secondary"
                  flow="add,save"
                  link="/companys"
                  type="button"
                >
                  Cancel
                </button>
              </a>
                 
              <button
                class="btn btn-primary"
                flow="save,add"
                type="submit"
              >
                Save
              </button>
                 
            </div>
          </div>
          <div
            class="error-banner d-flex align-items-center "
          >
            <div>
              <div
                class="error-heading"
              >
                OOPS!
              </div>
              <div
                class="error-reason"
              >
                Something went wrong! please try again.
              </div>
            </div>
            <div
              class="ml-auto"
            >
              <button
                class="btn btn-link"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-times fa-w-10 "
                  color="#212529"
                  data-icon="times"
                  data-prefix="fal"
                  focusable="false"
                  role="img"
                  style="font-size: 1.5rem;"
                  viewBox="0 0 320 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M193.94 256L296.5 153.44l21.15-21.15c3.12-3.12 3.12-8.19 0-11.31l-22.63-22.63c-3.12-3.12-8.19-3.12-11.31 0L160 222.06 36.29 98.34c-3.12-3.12-8.19-3.12-11.31 0L2.34 120.97c-3.12 3.12-3.12 8.19 0 11.31L126.06 256 2.34 379.71c-3.12 3.12-3.12 8.19 0 11.31l22.63 22.63c3.12 3.12 8.19 3.12 11.31 0L160 289.94 262.56 392.5l21.15 21.15c3.12 3.12 8.19 3.12 11.31 0l22.63-22.63c3.12-3.12 3.12-8.19 0-11.31L193.94 256z"
                    fill="currentColor"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="layout-content h-100 overflow-auto"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Mode of hours submission
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    class="btn-group"
                    name="modeOfHours"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="1"
                      type="button"
                    >
                      Pay Period
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="2"
                      type="button"
                    >
                      Annual
                    </button>
                  </div>
                </div>
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Mode of compensation
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    class="btn-group"
                    name="modeOfCompensation"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="1"
                      type="button"
                    >
                      Pay Period
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="2"
                      type="button"
                    >
                      Annual
                    </button>
                  </div>
                </div>
              </div>
              <div
                class=" sm form-group"
              >
                <label
                  class="form-label"
                >
                  Payroll Calendar Required?
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    class="btn-group"
                    name="isPayrollCalenderRequire"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="true"
                      type="button"
                    >
                      Yes
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="false"
                      type="button"
                    >
                      No
                    </button>
                  </div>
                </div>
              </div>
              <div
                class=" sm form-group"
              >
                <label
                  class="form-label"
                >
                  Census and payroll file generation required?
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    class="btn-group"
                    name="isFileGenerationRequire"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="true"
                      type="button"
                    >
                      Yes
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="false"
                      type="button"
                    >
                      No
                    </button>
                  </div>
                </div>
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Mode of Funding
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    class="btn-group"
                    name="modeOfFunding"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="1"
                      type="button"
                    >
                      ACH Credit/Wire
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="2"
                      type="button"
                    >
                      Debit
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
